website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx

```diff
@@ -2,77 +2,129 @@ import {
+            positioning elements.
           </DxcBulletedList.Item>
           <DxcBulletedList.Item>
-            Header and footer elements should be always present
+            When necessary use 4px to make more fine tuned adjustments.{" "}
           </DxcBulletedList.Item>
           <DxcBulletedList.Item>
-            Only use the mode overlay if the content cannot dynamically adapt to
-            the available space
+            Whenever possible, make sure that objects line up, both vertically
+            and horizontally.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            Align your bounding box to the grid, not the baseline of your text.
           </DxcBulletedList.Item>
         </DxcBulletedList>
       </>
     ),
   },
   {
-    title: "Sidenav: overlay and push",
+    title: "Breakpoints",
     content: (
       <>
-        <Figure caption="Overlay and push behavior of the sidenav component">
-          <Image
-            src={SidenavBehavior}
-            alt="Overlay and push behavior of the sidenav component"
-          />
-        </Figure>
         <DxcParagraph>
-          Overlay and push interactions by the sidenav component can alter the
-          way the layout is displayed. For overlay, the sidenav simply slides
-          over the content with no changes to position. The push interaction
-          moves the main container to the side depending on the width of the
-          sidenav.
+          Breakpoints define resolutions at which screen components adjust to
+          offer an optimal user experience across screen sizes and devices.
+          We&#39;ve defined five different breakpoints to accommodate multiple
+          web, tablet, and mobile screen resolutions:
+        </DxcParagraph>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th>Breakpoint</th>
+              <th>px</th>
+              <th>rem</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <Code>xsmall</Code>
+              </td>
+              <td>320</td>
+              <td>20</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>small</Code>
+              </td>
+              <td>480</td>
+              <td>30</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>medium</Code>
+              </td>
+              <td>720</td>
+              <td>45</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>large</Code>
+              </td>
+              <td>1056</td>
+              <td>66</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>xlarge</Code>
+              </td>
+              <td>1440</td>
+              <td>90</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <DxcParagraph>
+          The image below illustrates how we&#39;ve used data on the most
```

Comment:
Maybe change the code `&#39;` with a normal `'`.

Reply:
`&#39;` is the same as `'`, and we avoid the error that gives in Visual Studio.